examples/test_protected_charger.py

```diff
@@ -0,0 +1,73 @@
+import unittest
+
+from edg import *
+
+
+class ProtectedCharger(JlcBoardTop):
+    """A Lipo charger that does not blowup with reverse polarity from the battery
+    Key features:
+    - A type C Lipo charger
+    - Charging capable reverse polarity protection with PMOS
+    - A Port for load
+    """
+
+    def contents(self) -> None:
+        super().contents()
+
+        self.usb = self.Block(UsbCReceptacle(current_limits=(0, 3) * Amp))
+        self.vusb = self.connect(self.usb.pwr)
+
+        self.batt = self.Block(LipoConnector(actual_voltage=(3.7, 4.2) * Volt))
+
+        self.gnd_merge = self.Block(MergedVoltageSource()).connected_from(
+            self.usb.gnd, self.batt.gnd
+        )
+        self.gnd = self.connect(self.gnd_merge.pwr_out)
+        self.tp_gnd = self.Block(VoltageTestPoint()).connected(self.gnd_merge.pwr_out)
+
+        with self.implicit_connect(
+                ImplicitConnect(self.gnd, [Common]),
+        ) as imp:
+            (self.tp,), _ = self.chain(self.batt.pwr, self.Block(VoltageTestPoint()), )
```

Comment:
```suggestion
            self.tp = self.Block(VoltageTestPoint()).connected(self.batt.pwr)
```
Use consistent style?

Reply:
applied in be8264d